### PR TITLE
Remove CentOS 6 support

### PR DESCRIPTION
**Pull request by ramereth**
Now that EL6 has been EOL'd upstream, we should remove support from this cookbook. In addition, include the following
fixes:

- Update vault repo versions to latest
- MDL Fixes
- Cleanup metadata.rb formatting

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,42 +2,36 @@
...
 
-### Fixed
 - Fix Chef 12 compatibility (resolves #42)
-
-### Added
 - Kitchen suite for testing against Chef 12
 
 ## 4.0.1 (2020-07-20)
 
-### Added
```
Comment (tas50) [inline CHANGELOG.md:15]: why are all the headings removed?

Comment (ramereth) [reply CHANGELOG.md:15]: Figured it looked better without them in the end.

Comment (tas50) [reply CHANGELOG.md:15]: I wouldn't add these new, but we should keep the existing ones since they break down the changes into types nicely

**Review by tas50 — APPROVED** [review]
Stick the old changelog headings back and it's good to go as a major version release

**Pull request merged**